setup.py

```diff
@@ -618,7 +622,8 @@ def pxd(name):
     '_libs.window': {
         'pyxfile': '_libs/window',
         'pxdfiles': ['_libs/skiplist', '_libs/src/util'],
-        'language': 'c++'},
+        'language': 'c++',
+        'suffix': '.cpp'},
```

Comment:
rather than adding `'suffix'` any reason not to base the suffix on language='c++'?

Reply:
I guess I just saw it as more of a configuration piece that belonged in a dictionary over branches in the code.

I could do something like:

```python
    if suffix == '.pyx':
        source_suffix = '.pyx'
    elif data.get('language', 'c') == 'c++':
        source_suffix = '.cpp'
    else:
        source_suffix = '.c'
```

But that feels a bit verbose. Thoughts?

Reply:
Not a huge deal, but that feels cleaner too me.  Reading the config I would probably guess ``.suffix` controls the suffix of the output of cython, which it woudln't.